scheduledworkflow/tox.ini

```diff
@@ -3,6 +3,7 @@
 [tox]
 skipsdist = True
 skip_missing_interpreters = True
+envlist = unit, sanity, integration
```

Comment:
Same as [this](https://github.com/canonical/pipelines-rocks/pull/63#discussion_r1452357669)

Reply:
updated!